core/vibes/soul/sections/cart/shipping-form/index.tsx

```diff
@@ -0,0 +1,418 @@
+                onClick={() => setShowForms(!showForms)}
+              >
+                {addLabel}
+              </button>
+            )
+          )}
+        </div>
+
+        {state.shippingOption && (
+          <div className="flex gap-1.5 text-xs">
+            <span className="font-medium text-contrast-400">{state.shippingOption.label}</span>
+            {!showForms && (
+              <button
+                className="font-semibold focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-[var(--button-focus,hsl(var(--primary)))] focus-visible:ring-offset-2"
+                onClick={() => setShowForms(true)}
+              >
+                {changeLabel}
+              </button>
+            )}
+          </div>
+        )}
+      </div>
+
+      {showForms && (
+        <>
+          {showAddressForm ? (
+            <form {...getFormProps(addressForm)} action={formAction} className="space-y-5">
+              {Array.isArray(countries) ? (
+                <Select
+                  errors={addressFields.country.errors}
+                  key={addressFields.country.id}
+                  label={countryLabel}
+                  name={addressFields.country.name}
+                  onBlur={countryControl.blur}
+                  onFocus={countryControl.focus}
+                  onValueChange={countryControl.change}
+                  options={countries}
+                  placeholder=""
+                  value={countryControl.value}
+                />
+              ) : (
+                <Input
+                  {...getInputProps(addressFields.country, { type: 'text' })}
+                  errors={addressFields.country.errors}
+                  key={addressFields.country.id}
+                  label={countryLabel}
+                />
+              )}
+              <Input
+                {...getInputProps(addressFields.city, { type: 'text' })}
+                errors={addressFields.city.errors}
+                key={addressFields.city.id}
+                label={cityLabel}
+              />
+              <div className="flex gap-3">
+                {Array.isArray(states) ? (
+                  <Select
+                    disabled={addressFields.country.value === undefined}
+                    errors={addressFields.state.errors}
+                    key={addressFields.state.id}
+                    label={stateLabel}
+                    name={addressFields.state.name}
+                    onBlur={stateControl.blur}
+                    onFocus={stateControl.focus}
+                    onValueChange={stateControl.change}
+                    options={
+                      states.find((s) => s.country === addressFields.country.value)?.states ?? []
+                    }
+                    placeholder=""
+                    value={stateControl.value}
+                  />
+                ) : (
+                  <Input
```

Comment:
@jorgemoya After I add a shipping address and then try to go back and edit it, my `State` dropdown doesn't open:
![image](https://github.com/user-attachments/assets/8f0fc447-acdc-4bc8-9f78-4bd9532d9c80)

Reply:
This should be fixed now!